src/scss/utilities/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$colors: (
```

Comment:
There are no direct bootstrap overrides here. You are using your own theming system, not the Bootstrap theming system.